packages/babel-helper-replace-supers/src/index.js

```diff
@@ -227,7 +227,7 @@ export default class ReplaceSupers {
           t.variableDeclarator(ref, node.left),
         ]),
         t.expressionStatement(t.assignmentExpression("=", node.left,
-          t.binaryExpression(node.operator[0], ref, node.right))),
+          t.binaryExpression(node.operator.slice(0, -1), ref, node.right))),
```

Comment:
@aickin Changed this from `.substring(0, node.operator.length - 1)` since it's easier